code/modules/reagents/Chemistry-Reagents/Blob-Reagents.dm

```diff
@@ -22,87 +36,58 @@
-	if(iscarbon(M))
-		M.emote("scream")
-
-// Combo Reagents
-
-/datum/reagent/blob/skin_melter
-	name = "Skin Melter"
-	id = "skin_melter"
-	description = ""
-	color = "#7F0000"
-	message_living = ", and you feel your skin char and melt"
+/datum/reagent/blob/lexorin_jelly //does tons of oxygen damage and a little brute
+	name = "Lexorin Jelly"
+	id = "lexorin_jelly"
+	color = "#00FFC5"
+	message_living = ", and your lungs feel heavy and weak"
 
-/datum/reagent/blob/skin_melter/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
-	M.adjust_fire_stacks(round(reac_volume/12))
+/datum/reagent/blob/lexorin_jelly/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
 	reac_volume = ..()
 	M.apply_damage(0.4*reac_volume, BRUTE)
-	M.apply_damage(0.4*reac_volume, BURN)
-	M.IgniteMob()
-	if(iscarbon(M))
-		M.emote("scream")
+	M.apply_damage(1*reac_volume, OXY)
+	M.losebreath += round(0.3*reac_volume)
 
-/datum/reagent/blob/lung_destroying_toxin
-	name = "Lung Destroying Toxin"
-	id = "lung_destroying_toxin"
-	description = ""
-	color = "#00FFC5"
-	message_living = ", and your lungs feel heavy and weak"
+/datum/reagent/blob/kinetic //does semi-random brute damage
+	name = "Kinetic Gelatin"
+	id = "kinetic"
+	color = "#FFA500"
+	message = "The blob pummels you"
 
-/datum/reagent/blob/lung_destroying_toxin/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+/datum/reagent/blob/kinetic/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
 	reac_volume = ..()
-	M.apply_damage(0.8* reac_volume, OXY)
-	M.losebreath += round(0.6*reac_volume)
-	M.apply_damage(0.8*reac_volume, TOX)
+	var/damage = rand(5, 35)/25
+	M.apply_damage(damage*reac_volume, BRUTE)
 
-// Special Reagents
+/datum/reagent/blob/cryogenic_liquid //does low burn damage and stamina damage and cools targets down
+	name = "Cryogenic Liquid"
+	id = "cryogenic_liquid"
+	color = "#8BA6E9"
+	message = "The blob splashes you with an icy liquid"
+	message_living = ", and you feel cold and tired"
 
-/datum/reagent/blob/radioactive_liquid
-	name = "Radioactive Liquid"
-	id = "radioactive_liquid"
-	description = ""
-	color = "#00EE00"
-	message_living = ", and your skin feels papery and everything hurts"
-
-/datum/reagent/blob/radioactive_liquid/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
+/datum/reagent/blob/cryogenic_liquid/reaction_mob(mob/living/M, method=TOUCH, reac_volume)
 	reac_volume = ..()
-	M.apply_damage(0.4*reac_volume, BRUTE)
-	if(ishuman(M))
-		M.irradiate(1.6*reac_volume)
-		if(prob(1.3*reac_volume))
-			randmuti(M)
-			if(prob(98))
-				randmutb(M)
-			domutcheck(M, null)
-			updateappearance(M)
-
-/datum/reagent/blob/dark_matter
+	M.apply_damage(0.4*reac_volume, BURN)
+	M.adjustStaminaLoss(0.4*reac_volume)
+	M.reagents.add_reagent("frostoil", 0.4*reac_volume)
+
+/datum/reagent/blob/dark_matter //does brute damage and throws or pulls nearby objects at the target
 	name = "Dark Matter"
 	id = "dark_matter"
-	description = ""
-	color = "#61407E"
+	color = "#800080"
```

Comment:
Why did you make this the same color as Sorium? This needs to be reverted, the color of a blob is essential gameplay info.